File: Iniciante/1020 - Idade em Dias/Python/1020.py
def main():
    age_in_days = int(input())
    converted_time = convert_time(age_in_days)
    print(converted_time)


def convert_time(age_in_days):
    DAYS_IN_A_YEAR = 365
    DAYS_IN_A_MONTH = 30
    years, remaining_days = divmod(age_in_days, DAYS_IN_A_YEAR)
    months, days = divmod(remaining_days, DAYS_IN_A_MONTH)
    return f"{years} ano(s)\n{months} mes(es)\n{days} dia(s)"


if __name__ == "__main__":
    main()

File: Iniciante/1020 - Idade em Dias/JavaScript/1020.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const ageInDays = parseInt(input);
  const convertedTime = convertTime(ageInDays);
  console.log(convertedTime);
}

function convertTime(ageInDays) {
  const DAYS_IN_A_YEAR = 365;
  const DAYS_IN_A_MONTH = 30;
  const years = Math.trunc(ageInDays / DAYS_IN_A_YEAR);
  const remainingDays = ageInDays % DAYS_IN_A_YEAR;
  const months = Math.trunc(remainingDays / DAYS_IN_A_MONTH)
  const days = remainingDays % DAYS_IN_A_MONTH;
  return `${years} ano(s)\n${months} mes(es)\n${days} dia(s)`;
}

main();
